src/vm/arm/ehhelpers.S

```diff
@@ -99,6 +99,7 @@ OFFSET_OF_FRAME=(4 + SIZEOF__GSCookie)
         NESTED_ENTRY CallEHFunclet, _TEXT, NoHandler
 
         PROLOG_PUSH  "{r4-r11, lr}"
+        PROLOG_STACK_SAVE r7
         alloc_stack  4
 
```

Comment:
instead of this fix. I would experiment with the unwind annotations for alloc_stack & free_stack https://github.com/dotnet/coreclr/blob/master/src/pal/inc/unixasmmacrosarm.inc#L60 .  Maybe free_stack does not need .pad annotation